test/transport.test.js

```diff
@@ -46,15 +46,15 @@ describe('transport', function () {
       var listen = Transport.listen(_.noop)
       var seneca = {
         log: {
-          info: function () {
-
-          }
+          info: _.noop,
```

Comment:
seems unwise to redefine noop when it's already in lodash - better to be consistent I would say